README.md

```diff
@@ -1,4 +1,4 @@
-# Screenshot Tool
+# Screenshot
 [![Translation status](https://l10n.elementary.io/widgets/screenshot-tool/-/svg-badge.svg)](https://l10n.elementary.io/projects/screenshot-tool/?utm_source=widget)
 
 ![Screenshot Tool Screenshot](data/screenshot.png?raw=true)
```

Comment:
```suggestion
![Screenshot](data/screenshot.png?raw=true)
```

Removing "Tool" results "Screenshot Screenshot", which is funny, so just "Screenshot" would be enough.